packages/admin/blocks-admin/src/blocks/factories/createListBlock.tsx

```diff
@@ -333,35 +359,39 @@ export function createListBlock<T extends BlockInterface, AdditionalItemFields e
+                                                                                title={
+                                                                                    isMaxVisibleBlocksMet && !data.visible ? (
                                                                                         <FormattedMessage
                                                                                             id="comet.blocks.list.maxVisibleBlocks"
-                                                                                            defaultMessage="Max visible blocks: {maxVisibleBlocks}"
+                                                                                            defaultMessage="Max. visible blocks: {maxVisibleBlocks}"
                                                                                             values={{ maxVisibleBlocks }}
                                                                                         />
-                                                                                    }
-                                                                                >
-                                                                                    <span>
-                                                                                        <IconButton disabled size="small">
-                                                                                            {data.visible ? (
-                                                                                                <Visible color="secondary" />
-                                                                                            ) : (
-                                                                                                <Invisible color="action" />
-                                                                                            )}
-                                                                                        </IconButton>
-                                                                                    </span>
-                                                                                </Tooltip>
-                                                                            )
+                                                                                    ) : !isMinVisibleBlocksMet ||
+                                                                                      (minVisibleBlocks === totalVisibleBlocks && data.visible) ? (
+                                                                                        <FormattedMessage
+                                                                                            id="comet.blocks.list.minVisibleBlocks"
+                                                                                            defaultMessage="Min. visible blocks required: {minVisibleBlocks}"
```

Comment:
It seems strange to me that the min message has a "required", but the max message has no "allowed" 🤔

Reply:
You're right. I'll update that 👍🏻